internal/xds/bootstrap/tlscreds/bundle_ext_test.go

```diff
@@ -253,3 +339,97 @@ func (s) TestMTLS(t *testing.T) {
 		t.Errorf("EmptyCall(): got error %v when expected to succeed", err)
 	}
 }
+
+func (s) TestMTLSSPIFFE(t *testing.T) {
+	s := stubserver.StartTestService(t, nil, grpc.Creds(testutils.CreateServerTLSCredentialsCompatibleWithSPIFFE(t, tls.RequireAndVerifyClientCert)))
+	defer s.Stop()
+
+	cfg := fmt.Sprintf(`{
+		"certificate_file": "%s",
+		"private_key_file": "%s",
+		"spiffe_trust_bundle_map_file": "%s"
+	}`,
+		testdata.Path("spiffe_end2end/client_spiffe.pem"),
+		testdata.Path("spiffe_end2end/client.key"),
+		testdata.Path("spiffe_end2end/client_spiffebundle.json"))
```

Comment:
I'm not convinced about this formatting. Please check go/go-style/decisions#indentation-confusion